src/legacy/core_plugins/visualizations/public/np_ready/public/plugin.ts

```diff
@@ -18,7 +18,33 @@
  */
 import { PluginInitializerContext, CoreSetup, CoreStart, Plugin } from 'src/core/public';
 import { TypesService, TypesSetup, TypesStart } from './types';
-import { setUISettings, setTypes, setI18n } from './services';
+import {
+  setUISettings,
+  setCapabilities,
+  setTypes,
+  setHttp,
+  setSavedObjectsClient,
+  setI18n,
+} from './services';
+import { ExpressionsSetup } from '../../../../../../plugins/expressions/public';
+import { IEmbeddableSetup } from '../../../../../../plugins/embeddable/public';
+
+import { visualization as visualizationFunction } from '../../expressions/visualization_function';
+import { visualization as visualizationRenderer } from '../../expressions/visualization_renderer';
+import { VisualizeEmbeddableFactory } from '../../embeddable/visualize_embeddable_factory';
+import { VISUALIZE_EMBEDDABLE_TYPE } from '../../embeddable';
+import { showNewVisModal } from '../../wizard';
```

Comment:
//nit 
Not sure why the plugin imports things outside the np_ready folder?

Reply:
as those are not np_ready yet, will move them in a followup, trying to keep PR's "small" :)